defs/scenarios/juju/charms_bugs.yaml

```diff
@@ -14,3 +20,15 @@ conclusions:
         failed since the disk has an LVM header. If you are sure the disk is
         not in active use by LVM, you may run pvremove on it before using it as
         an OSD - see LP bug for details.
+
+  lp1812361:
+    decision: has_1812361
+    raises:
+      type: LaunchpadBug
+      bug-id: 1812361
+      message: >-
+        known glance-simplestream-sync bug identified - this happens because
+        the relation data received from keystone does not have the expected
+        data. Upgrading charm-keystone to version >= stable/539 should fix the
```

Comment:
Can you add some information to the message about what information is missing from the relation that is expected to exist?

Reply:
Done